self-hosted/backup-and-restore/logical-backup.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Logical backup with pg_dump and pg_restore
+excerpt: Back up and restore a hypertable or an entire database using native PostgreSQL commands
+keywords: [backups, restore]
+tags: [recovery, logical backup, pg_dump, pg_restore]
+---
+
+# Logical backup with `pg_dump` and `pg_restore`
+
+You backup and restore each self-hosted PostgreSQL database with TimescaleDB enabled using the native 
+PostgreSQL [`pg_dump`][pg_dump] and [`pg_restore`][pg_restore] commands. This also works for compressed hypertables, 
+you don't have to decompress the chunks before you begin.
+
+If you are using `pg_dump` to backup regularly, make sure you keep
+track of the versions of PostgreSQL and TimescaleDB you are running. For more
+information, see [Versions are mismatched when dumping and restoring a database][troubleshooting-version-mismatch].
+
+This page shows you how to:
+
+- [Back up and restore an entire database][backup-entire-database]
+- [Back up and restore individual hypertables][backup-individual-tables]
+
+You can also [upgrade between different versions of TimescaleDB][timescaledb-upgrade]. 
+
+## Prerequisites
+
+- A source database to backup from, and a target database to restore to.
+- Install the PostgreSQL client tools on your migration machine.
+
+   This includes `psql`, and `pg_dump`.
+
+## Back up and restore an entire database
+
+You backup and restore an entire database using `pg_dump` and `psql`. in Terminal: 
+
+<Procedure>
+
+In Terminal:
```

Comment:
"In terminal" is repeated. Also, let's use keep "terminal" lower case.

Reply:
Dearie me, thank you. Updated.